src/components/charts/publications/disciplines/dynamique-ouverture/chart-taux-ouverture.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable react/no-this-in-sfc */
+import { Col, Container, Row } from '@dataesr/react-dsfr';
+import Highcharts from 'highcharts';
+import HCExportingData from 'highcharts/modules/export-data';
+import HCExporting from 'highcharts/modules/exporting';
+import HighchartsReact from 'highcharts-react-official';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useIntl } from 'react-intl';
+
+import { discipline100 } from '../../../../../style/colours.module.scss';
+import { getGraphOptions } from '../../../../../utils/helpers';
+import useGlobals from '../../../../../utils/Hooks/useGetGlobals';
+import Loader from '../../../../Loader';
+import GraphComments from '../../../graph-comments';
+import GraphTitle from '../../../graph-title';
+import useGetData from './get-data';
+
+HCExporting(Highcharts);
+HCExportingData(Highcharts);
+
+const Chart = ({ graphComments }) => {
+  const intl = useIntl();
+  const graphId = 'app.sante-publi.disciplines.dynamique-ouverture.chart-taux-ouverture';
```

Comment:
To add to fr.json and en.json

Reply:
C'est noté, seul le fr.json est ok pour le moment